js/plugins/ollama/src/embeddings.ts

```diff
@@ -76,12 +79,12 @@ export function defineOllamaEmbedder(
       },
     },
     async (input, config) => {
-      const serverAddress = config?.serverAddress || options.serverAddress;
```

Comment:
in the v2 plugin system, the config parameter in the embedder action handler doesn't seem to have a serverAddress property